### ISSUE TITLE
NTR: immediate type sensitivity to drug

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3794]
New term request: immediate type sensitivity to drug. Definition: \
Source: \
Parent: \
Disease association:

**Comment by LCCarmody** [comment]
- [ ] Term label: "Immediate-type hypersensitivity drug reactions"
- [ ] Definition: " Hypersensitivity that is observed within 1 hr of exposures where hypersensitivity is broadly defined as ‘objectively reproducible symptoms or signs, initiated by exposure to a defined stimulus at a dose tolerated by normal subjects’ and may be caused by immunologic (allergic) and non-immunologic mechanisms "
- [ ] Source: PMID: 24286446 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4168374/
- [ ] Parent: Drug sensitivity (Need NTR)? Not sure where this term would go.  Maybe "Abnormality of metabolism/homeostasis"   https://hpo.jax.org/app/browse/term/HP:0001939 _AND_ Immunologic hypersensitivity https://hpo.jax.org/app/browse/term/HP:0100326 ?
- [ ] Should we also add the term "Late onset hypersensitivity" with the same source and parents.  
         - [ ] definition "reactions are often defined as those occurring >72 h after drug administration and these are usually T-cell-mediated and generally lack the rapidly evolving life-threatening features that are typical of anaphylaxis"

**Comment by pnrobinson** [comment]
Let's hold off on the late onset until feedback from the immunologists is available. It seems there is a lot of inference and I am uncertain how to classify the phenotypes. The immediate type seems to be well defined.
